llvm/cmake/modules/HandleLLVMOptions.cmake

```diff
@@ -1240,7 +1240,7 @@ set(LLVM_ENABLE_EXPERIMENTAL_DEPSCAN OFF CACHE BOOL
 set(LLVM_DEPSCAN_MODE "" CACHE STRING "Mode for -fdepscan if used")
 set(LLVM_DEPSCAN_DAEMON "" CACHE STRING "Path to existing DepScan daemon to use")
 set(LLVM_CAS_BUILTIN_PATH "" CACHE STRING "Path to pass for -fcas-builtin-path")
-set(LLVM_CAS_BUILTIN_PATH_Default "/^llvm::cas::builtin::default/llvm.cas.builtin.default")
+set(LLVM_CAS_BUILTIN_PATH_Default "-")
```

Comment:
"auto"?

Reply:
Reverted in 7756976fa6d4cf23fbb7f124ba95003add6a106a since it gets passed to lld.